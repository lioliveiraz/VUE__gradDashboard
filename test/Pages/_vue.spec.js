import _vue from '../../pages/_.vue';
import { validateTruthiness } from './../utils/index';
import { shallowMount } from '@vue/test-utils';


describe('<_vue/>', () => {
    let wrapper;

    beforeEach(async () => {
        wrapper = await shallowMount(_vue, {
            mocks: {
                $router: []
            },

        });
    });
    afterEach(() => {
        if (wrapper) {
            wrapper.destroy();
        }
    });
    it('should render correctly', () => {
        validateTruthiness(wrapper.get(".g-login-box"));
        expect(wrapper).toMatchSnapshot();

    });

});