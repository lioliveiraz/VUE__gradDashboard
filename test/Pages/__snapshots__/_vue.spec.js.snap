// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<_vue/> should render correctly 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
